src/main/scala/io/iohk/ethereum/App.scala

```diff
@@ -35,21 +35,17 @@ object App {
 
     server ! ServerActor.StartServer(NetworkConfig.Server.listenAddress)
 
-    val bootstrapNodes = NetworkConfig.Discovery.bootstrapNodes.map(new URI(_))
-    bootstrapNodes.foreach { node =>
-      peerManager ! PeerManagerActor.ConnectToPeer(node)
-      //TODO change to CLI command?
-      Thread.sleep(2 * 1000)
-      val storage = FastSyncActor.Storage(
-        new BlockHeadersStorage(EphemDataSource()),
-        new BlockBodiesStorage(EphemDataSource()),
-        new ReceiptStorage(EphemDataSource()),
-        new MptNodeStorage(EphemDataSource()),
-        new EvmCodeStorage(EphemDataSource())
-      )
-      peerManager ! PeerManagerActor.StartFastDownload(node,
-        ByteString(Hex.decode("12d7b70f28b819867087c37e4190b727dfbb1f7f34e2687e5ad126a31cf051be")), storage)
-    }
+    val dataSource = EphemDataSource()
+    val fastSyncController = actorSystem.actorOf(FastSyncController.props(
+      peerManager,
+      nodeStatusHolder,
+      new MptNodeStorage(dataSource),
+      new BlockHeadersStorage(dataSource),
+      new BlockBodiesStorage(dataSource),
+      new ReceiptStorage(dataSource),
+      new EvmCodeStorage(dataSource)), "fast-sync-controller")
+
+    fastSyncController ! FastSyncController.StartFastSync(ByteString(Hex.decode("81e2dcb132c2af3cb84591466aa904bb054f0b9ba52e369c06a271f6d92190db")))
```

Comment:
I've been digging around other implementations regarding this and it's definitely for another task